ansible/library/minigraph_facts.py

```diff
@@ -531,6 +531,9 @@ def parse_xml(filename, hostname):
     elif hwsku == "Celestica-E1031-T48S4":
         for i in range(1, 53):
             port_alias_to_name_map["etp%d" % i] = "Ethernet%d" % ((i - 1))
+    elif hwsku == "et6448m":
+        for i in range(0, 52, 1):
```

Comment:
Any particular reason for using range(0, 52, 1) instead of range(0, 52)

Reply:
Hi @sumukhatv 

     step=1 is not required,  i will change it

Thanks
Rajneesh

Reply:
Removed it. Please review the changes

Thanks
Rajneesh